src/index.ts

```diff
@@ -32,7 +33,7 @@ async function storePathRelativeToHome (pkgRoot: string, relStore: string) {
   await mkdirp(path.dirname(tempFile))
   await touch(tempFile)
   const homedir = getHomedir()
-  if (await canLink(tempFile, path.join(homedir, path.basename(tempFile)))) {
+  if (await canLink(tempFile, path.join(homedir, uuid()))) {
```

Comment:
you should be able to use `pathTemp(homedir)` instead of `path.join(homedir, uuid())`